lutris/gui/widgets/game_bar.py

```diff
@@ -101,14 +102,25 @@ def get_game_name_label(self):
     def get_runner_button(self):
         icon_name = self.game.runner.name + "-symbolic"
         runner_icon = Gtk.Image.new_from_icon_name(icon_name, Gtk.IconSize.MENU)
-        runner_icon.show()
```

Comment:
I didn't see any visible negative effect of removing this. Not sure if it's actually unnecessary or I just don't know what to look for.